debian/copyright

```diff
@@ -0,0 +1,47 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
This file shouldn't be necessary, it's referencing files in citus which are unused in azure_gdpr

Reply:
Given that this comment is not resolved by, wanted to make sure that we're ok with adding this file.

Indeed, I'm not much convinced about whether it's ok to use citus's copyright, so wondering what @hanefi would say.

Reply:
We had an internal discussion and decided to go with a license similar to that of enterprise packages at https://github.com/citusdata/packaging/blob/all-enterprise/debian/copyright